meetings/agenda.md

```diff
@@ -12,3 +12,32 @@ Please file pull requests to add, or discuss items to add, to the agenda.
 [Maximilian Bösing](https://github.com/boesing) wants to drop the line length limit from the `laminas-codestandard` ruleset.
 
 The line length limit is counter-productive in combination with psalm and more precise argument-/return-type definitions.
+
+### Allow PHP-Assert along with Webmozart-Assert in Specific Cases
+
+[Maximilian Bösing](https://github.com/boesing) wants to talk about the usage of `assert` (PHP) and `Assert` (Webmozart) when providing types for psalm.
+In some cases, having runtime assertions may consume unnecessary opcodes just to provide proper psalm-types.
+
+Lets assume the following code:
+
+```php
+$foo = sprintf('foo bar baz %d', 1);
+assert($foo !== '');
+```
+
+Psalm is not able to verify that `$foo` is a non-empty-string, even tho, the first argument is a non-empty string and therefore, the return value must either be `false` (in case of an error) or `non-empty-string`.
+
+https://psalm.dev/r/add6e2b321
+
+I prefer using `assert($foo !== '')` over `Assert::stringNotEmpty` to avoid redundant runtime checks which actually only have to be made to "make psalm happy".
```

Comment:
> only have to be made to "make psalm happy".

Your system will crash much later due to invalid data: `assert()` is for things that are considered "known" invariants, not for verification of input, and only works for constant bits.